inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/TenantRoleServiceImpl.java

```diff
@@ -116,4 +119,17 @@ public TenantRoleInfo getByUsernameAndTenant(String name, String tenant) {
         return CommonBeanUtils.copyProperties(entity, TenantRoleInfo::new);
     }
 
+    @Override
+    public List<String> listTenantByUser(String user) {
+        TenantRolePageRequest request = new TenantRolePageRequest();
+        request.setPageSize(1);
```

Comment:
Why sizd == 1?

Reply:
Because this method try to list all tenant roles.